docs/integrations/unity-catalog-spark.md

```diff
@@ -0,0 +1,179 @@
+|             3|air conditioner|  
++--------------+---------------+
+```
+
+Now, let’s look at how to create external Unity Catalog tables.
+
+## Create an external Unity Catalog table with Spark
+
+Here’s how to create an external Unity Catalog table with Spark:
+
+```python
+letters = spark.createDataFrame([  
+    (1, "a"),   
+    (2, "b"),  
+    (3, "c"),  
+], ["id", "letter"])
+
+letters.write.format("parquet").saveAsTable("stores.whatever.letters", path="/tmp/letters")
+```
+
+When you manually specify the path, as in the example above, Spark creates an unmanaged table.
+
+Let’s look at a visualization of our current tables:
+
+![UC Spark 2](../assets/images/uc_spark2.png)
+
+See this page to learn more about the difference between external and managed Unity Catalog tables.
+
+## Spark write data to path
+
+You can also use Spark to write data to a path and not make any Unity Catalog entries.
+
+```python
+people = spark.createDataFrame([  
+    (1, "li"),   
+    (2, "chung"),  
+], ["id", "first_name"])
+
+people.write.format("parquet").save("/tmp/people")
+```
+
+As you can see in the example above, writing data with Spark doesn’t necessarily create a new entry in the Unity Catalog. This writes data to a path but doesn’t create any associated entries in the Unity Catalog, burdening the user with managing the credentials and access to the data themselves.
+
+## Advantage of Unity Catalog for Spark
+
+Unity Catalog has many advantages for Spark users.
+
+**Decoupling data storage location from business logic**
+
+You don’t want to hardcode file paths in your code because then the location of your data becomes coupled with your business logic.  Here’s an example of bad code:
+
+```python
+people = spark.read.format("parquet").load("/tmp/people")  
+...
+```
+
+This code would break if the location of the underlying data ever changes.
+
+```python
+people = spark.table("stores.us_east.people")
+```
+
+This allows for the underlying table to be moved without breaking the business logic.
+
+**Making it easier to organize and find datasets**
+
+When datasets are registered in Unity Catalog, you can use the CLI or UI to easily find all the data assets.  Let’s take a look at the Unity Catalog UI:
+
+![UC Spark 3](../assets/images/uc_spark3.png)
+
+We can easily see the schemas, tables, volumes, and functions that are registered in Unity Catalog!
+
+**Allows for ergonomic implementations of common design patterns, like the Medallion architecture**
+
+The Medallion architecture is a conceptual data organization framework that’s sometimes useful when building ETL pipelines, as described in [this blog post](https://delta.io/blog/delta-lake-medallion-architecture/).
+
+Unity Catalog is a great way to organize data assets for pipelines that use the Medallion architecture.  You can create bronze, silzer, and gold schemas to clearly bucket the different data assets.
+
+**Flexibility to use managed or external tables**
```

Comment:
lets not do this right now.. since managed tables creates and writes are not supported yet.